Lib/email/utils.py

```diff
@@ -42,6 +42,8 @@
 
 specialsre = re.compile(r'[][\\()<>@,:;".]')
 escapesre = re.compile(r'[\\"]')
+realname_comma_re = re.compile(r'"[^"]*,[^"]*"')
```

Comment:
```suggestion
realname_comma_re = re.compile(r'"[^",]*+,[^"]*+"')
```
It is faster. But I am not sure that the use of such regex is correct.